waiter/src/waiter/state/responder.clj

```diff
@@ -478,13 +479,21 @@
        :response-chan response-chan
        :response response-code})))
 
+(defn- update-in-with-log-instance
+  [mp log-body event-type log-level key-sequence func & args]
```

Comment:
Based on the single invocation below isn't `log-body` always the `instance-id`?
We should avoid the use of `apply` whenever we can.

Reply:
I added the abstract layer so that we can reuse this function later. Should I make it only consider the current case and we would change it later if we would need to?

Reply:
Yes, please change it to consider the current case.

Reply:
Done